### PR TITLE
Fix : Better product image cover upload error message

### PR DESCRIPTION
**Pull request by vibby**

### CODE REVIEW

File: application/controllers/Products.php [hunk 1]
```diff
@@ -143,12 +143,13 @@ public function save() {
 		if( ( array_key_exists( "id", $data ) && $_FILES["cover_image"]["name"] ) || $is_new ) {
 			$cover_image = $this->_uploadCoverImage();
 			if( ! $cover_image ) {
-				$this->output->set_output( json_encode( [ "error" => "Error uploading cover image" ] ) );
+				$this->output->set_output( json_encode( [ "error" => $this->imageUploadError ?? "Error uploading cover image" ] ) );
```
Comment (akjackson1) [inline application/controllers/Products.php]: Shouldn’t we move this check to the “if” statement in the line above?

Comment (vibby) [reply application/controllers/Products.php]: The idea is to use the error message if defined, or to fallback on «Error uploading cover image» if no specific message is given

Comment (akjackson1) [reply application/controllers/Products.php]: Oops I guess I was multitasking when commented 😆. Looks good!

Comment (vibby) [reply application/controllers/Products.php]: You’re welcome ;)

**Pull request merged**